src/UI/Rest/Controller/ResarchStation/ResearchStationCommandController.php

```diff
@@ -38,8 +41,10 @@ public function changeAngle(Request $request): Response
     public function reportRequest(Request $request): Response
     {
         $destination = $request->query->get('destination');
-        $command = new ReportARequestCommand($destination);
+        $this->handleWithDelay(new ReportARequestCommand($destination));
+
+        $response = new ApiResponse();
 
-        $this->handleWithDelay($command);
+        return $response->setStatusCode(200);
```

Comment:
```suggestion
        return new Response([], 200);
```

Tak wyglądają defaultowe wartosci na Response.